ERCS/erc-7786.md

```diff
@@ -20,9 +20,9 @@ Cross-chain messaging protocols (or bridges) allow communication between smart c
 
 Because almost every protocol implements a different workflow using a specific interface, portability between bridges is currently basically impossible. This also prevents the development of generic contracts that rely on cross chain communication.
 
-The objective of this ERC is to provide a standard interface, and a corresponding workflow, for performing cross-chain communication between contracts. Existing cross-chain communication protocols that do not natively implement this interface should be able to adopt it using adapter gateway contracts.
+The objective of this ERC is to provide a standard interface, and corresponding workflows, for performing cross-chain communication between contracts. Existing cross-chain communication protocols that do not natively implement this interface should be able to adopt it using adapter gateway contracts.
 
-Compared to previous ERCs in this area, this ERC offers compatibility with chains outside of the Ethereum/EVM ecosystem, and it is extensible to support the different feature sets of various protocols while offering a shared core of standard functionality.
+Compared to previous ERCs in this area, this ERC offers compatibility with chains outside of the Ethereum/EVM ecosystem, and it is extensible to support the different feature sets of various protocols while offering a shared core of standard functionality. Moreover, this standard supports both PUSH and PULL communication modes, capturing a larger class of interoperability protocols.
```

Comment:
I actually have concerns about using push and pull in this way, based on our conversations in the working group. I think these terms are too broad, and mean different things to different people, and so adding this here could lead to confusion.

Reply:
I agree, also because PUSH/PULL communication modes are not defined in this section. The sentence was removed in 39159f76.